.github/workflows/ci-cd.yml

```diff
@@ -107,10 +107,18 @@ jobs:
     strategy:
       matrix:
         arch: [armv7hf, aarch64]
+    outputs:
+      TAG: ${{ steps.vars.outputs.TAG }}
```

Comment:
TAG is not used in this job(?) - remove it

Reply:
The TAG variable is saved from "Set output" on row 116, but maybe this should be moved to the job "create_prerelease"? Since that's where it's actually needed.

Reply:
Since it is not used in this job remove it here and update to get the value of it in the `Create prerelease`job instead.
Alternatively, ${{ steps.meta_test.outputs.version }}, is the tag so you could set TAG to that here and only remove the `vars`step